### ISSUE TITLE
Train Precision low

### ISSUE THREAD

**Issue by remeberWei** [mikel-brostrom/Yolov5_StrongSORT_OSNet#442]
### Search before asking

- [X] I have searched the Yolov5_StrongSORT_OSNet [issues](https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/issues) and found no similar bug report.

### Question

Hello，I'm sorry to bother you. I have a question. When using deepsort v3 to train my own data set, the accuracy of the training set and test set is very low, only about 10%. Do you know the reason? Thank you

**Comment by mikel-brostrom** [comment]
What are you training? Yolo or he deep appearance descriptor in DeepSORT? What are you evaluating on? Are you using `eval.sh`? With this little information I cannot help you, sorry

**Comment by remeberWei** [comment]
> What are you training? Yolo or he deep appearance descriptor in DeepSORT? What are you evaluating on? Are you using `eval.sh`? With this little information I cannot help you, sorry

I'm sorry I didn't explain my reasons clearly. I want to train my own dataset through deepsort. Here are my steps: 

1. first, deduct the detection target in the image with code. As our data set, there is only one detection target. , but there are different categories, one picture is one category, and the detection targets in a total of 588 pictures are extracted, that is to say, there are 588 categories. 
2. Second, move train and test to deep_sort/deep directory, 
3. Then modify the categories in model.py. This category is 588 categories. The modify location as follows: https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/842959fffd5d894c9d22d02898632d0f5003621c/deep_sort_pytorch/deep_sort/deep/model.py#L55
4.Start training. The results after training for 40 epochs are as follows, showing that the accuracy is very low. I don’t know why, but my training method Problems? Looking forward to your reply, thank you very much.

![image](https://user-images.githubusercontent.com/91881703/176587577-30a542fb-5295-4c9a-ab79-67f40d649961.png)

**Comment by mikel-brostrom** [comment]
Sorry. torchreid is not my repository. I recommend you to post this question here: https://github.com/KaiyangZhou/deep-person-reid
